libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewListener.kt

```diff
@@ -120,4 +120,14 @@ abstract class NavigationViewListener {
      * Called when the info panel behavior updates to collapsed.
      */
     open fun onInfoPanelCollapsed() = Unit
+
+    /**
+     * Called when the info panel behavior updates to dragging.
+     */
+    open fun onInfoPanelDragging() = Unit
```

Comment:
You should update current.txt.

Reply:
☝️ 
I'll take care of it in my PR.
I am surprised that the static analysis check didn't detect this.

Reply:
Same here: https://github.com/mapbox/mapbox-navigation-android/pull/6254#discussion_r959949092